onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -67,17 +64,10 @@ def augment_graph(self):
                                    (node.name not in self.black_nodes)) or (node.name in self.white_nodes)
             if should_be_calibrate:
                 for input_tensor_name in node.input:
-                    if input_tensor_name in value_infos.keys():
```

Comment:
It will looks simple with:
tensors_to_calibrate .update(node.input)
tensors_to_calibrate.update(node.output)